app/api/deps/domain.py

```
@@ -25,3 +27,8 @@ def get_financial_institution_service(db: AsyncSession = Depends(get_db)):
 def get_account_service(db: AsyncSession = Depends(get_db)):
     repo = AccountRepo(db)
     return AccountService(repo)
+
+
+def get_currrency_service(db: AsyncSession = Depends(get_db)):
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)